src/Sulu/Bundle/AdminBundle/Resources/js/components/containerSizeAware/__mocks__/containerSizeAware.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import React from 'react';
+
+export default function containerSizeAware(Component: ReactClass<*>) {
```

Comment:
I often see the pattern that these HOCs are called `withSomething`. Following this I would name it `withContainerSize`, instead of introducing a new convention.